indigo_app/templates/indigo_api/_task_list_all.html

```diff
@@ -0,0 +1,55 @@
+{% load humanize indigo_app %}
+
+{% if tasks %}
+<table class="table table-striped table-bordered">
+  <thead>
+    <tr>
+      <th scope="col">Task</th>
+      <th scope="col">Place</th>
+      <th scope="col">Work/Document</th>
+    </tr>
+  </thead>
+  <tbody>
+  {% for task in tasks %}
+  {% if task.state != 'pending_review' %}
+    <tr>
+      <td>
+        <div class="task-list-span">
```

Comment:
Bootstrap has good flex utility classes. This would be `d-flex align-items-center`

Reply:
Also, I suspect we'll have to do something here so that the titles line up when the numbers are different lengths (eg. task #12 vs #12345). Eg a second column? Or wrap the icon, checkbox and number in a div and use flexbox.

Reply:
I forgot about the bootstrap utility classes. 🙈 Added them.

Reply:
Available tasks:
![Screenshot 2019-05-30 at 15 12 18](https://user-images.githubusercontent.com/13785692/58635126-58315980-82ed-11e9-9796-ba38597ced0f.png)

Task list:
![Screenshot 2019-05-30 at 15 13 24](https://user-images.githubusercontent.com/13785692/58635177-7eef9000-82ed-11e9-8dc0-4534de3ef47f.png)

@longhotsummer thoughts on this layout?

Reply:
I think it's important to keep the number, icon and title all on the same line, otherwise we lose the benefit of the columnar layout. If the title and labels are in their own column, then they'll wrap independently of the number and the icon.

```
|------------------|--------------...
| [ ] (icon) #1234 | Title label label label   |
|------------------|--------------...
```

Reply:
Makes sense. I'll update this.